ExampleApps/SwiftUIExample/Views/Design/Colors.swift

```diff
@@ -0,0 +1,41 @@
+//  swiftlint:disable:this file_name
+//  Colors.swift
+//  Colors
+//
+//  Created by Richard Gist on 8/25/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+extension Color {
+    static var primaryBackground: Color {
+        .black
+    }
+
+    static var primaryText: Color {
+        white
+    }
+
+    static var card: Color {
+        Color(red: 86 / 255, green: 86 / 255, blue: 118 / 255)
```

Comment:
Thought we were gonna extract this into a new initializer that already did the / 255 thing.